pkg/registry/thirdpartyresourcedata/codec.go

```diff
@@ -34,6 +34,7 @@ import (
 	"k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
```

Comment:
Is there an upstream PR we can track which contains these fixes? If so, can you add it to the commit -  this makes tracking our backported fixes between releases easier.